app/vmui/packages/vmui/src/api/explore-metrics.ts

```diff
@@ -0,0 +1,10 @@
+import { TimeParams } from "../types";
+
+export const getJobsUrl = (server: string, period: TimeParams): string =>
+  `${server}/api/v1/label/job/values?start=${period.start}&end=${period.end}`;
+
+export const getInstancesUrl = (server: string, period: TimeParams, job: string): string =>
+  `${server}/api/v1/label/instance/values?match[]={job="${job}"}&start=${period.start}&end=${period.end}`;
```

Comment:
Should we use strict equality or there can be `{job=~"some_.*"}`?

Reply:
@valyala, Could you look into this?

Reply:
It is better to use strict equality here. Also the `${job}` arg must be properly escaped before placing it inside doubleqoutes. Something like `${encodeURIComponent(JSON.stringify(job))}` should work. The same applies to any strings, which are passed to MetricsQL series selectors and to query string args.

Reply:
@Loori-R , it looks like you didn't address possible escaping issues for strings as described in the previous comment. Examples:
* If `job` equals to `foo&bar`, then this will break the url, e.g. it will look like `.../values?match[]={job="foo&bar"}&start=...`, so the `match[]` query arg will be parsed as `{job="foo` instead of `{job="foo&bar"}` on the server side. So the `match[]` query arg must be encoded with `encodeURLComponent()`.
* If `job` equals to `foo"bar`, then this will break the label filter, e.g. it will look like `{job="foo"bar"}`, which is incorrect. The `"` inside the `job` string must be properly escaped with `JSON.stringify()`.

Reply:
@valyala, oh, sorry, I missed it. Already fixed